dev-docs/prebid-1.0-API.md

```diff
@@ -17,6 +17,23 @@ This document describes the changes to the Publisher API for version 1.0.
 * TOC
 {:toc}
 
+## Deprecations 
+The following functions/variables are removed as part of 1.0.
+
+- All `pbjs._*` varaibles will be depracated as part of 1.0. This includes the following
+  - `pbjs._winningBids`
+  - `pbjs._bidsReceived`
+  - `pbjs._bidsRequested`
+  - `pbjs._adUnitCodes`
+  - `pbjs._adsReceived`
+  - `pbjs.cbTimeout `
+- `pbjs.addCallback` & `pbjs.removeCallback` in favor of [onEvent API](http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.onEvent). 
+- `pbjs.allBidsAvailable`
+- `pbjs.buildMasterVideoTagFromAdserverTag` in favor of `pbjs.adServers.dfp.buildVideoUrl`
+- `adUnit.sizeMapping` - see below for [details on sizeConfig]({{site.baseurl}}/dev-docs/prebid-1.0-API.html#size-mapping-changes) 
+
```

Comment:
`$$PREBID_GLOBAL$$.bidderTimeout` - Moved to config

https://github.com/prebid/Prebid.js/pull/1633 PR is not merged but it removes 
`$$PREBID_GLOBAL$$._adsReceived` (You already added that)
`$$PREBID_GLOBAL$$.timeoutBuffer` - Moved to config
`$$PREBID_GLOBAL$$.logging` - Moved to config
`$$PREBID_GLOBAL$$.publisherDomain` - Moved to config
`$$PREBID_GLOBAL$$.setBidderSequence` - Moved to config
`$$PREBID_GLOBAL$$.setS2SConfig` - Moved to config

Reply:
@jaiminpanchal27 what is `$$PREBID_GLOBAL$$.bidderTimeout` used for? Shouldn't this just be set in requestBids?

Reply:
I added `pbjs.timeoutBuffer` but the rest are covered  in the `pbjs.setConfig` section below this.